sparksteps/__main__.py

```diff
@@ -106,6 +107,7 @@ def create_parser():
     parser.add_argument('--num-task', type=int)
     parser.add_argument('--release-label', required=True)
     parser.add_argument('--s3-bucket', required=True)
+    parser.add_argument('--s3-path', default='')
```

Comment:
Change default to `'sparksteps/sources/'`

Reply:
Also name it something more explicit like `sources-path`

Reply:
decided to keep s3-path as discussed as it's now the root of where sparksteps writes assets and logs.